docs/tests/manual-scenarios.md

```diff
@@ -0,0 +1,93 @@
+
+**Czynności przygotowawcze**: -
+
+**Czynności końcowe**: Usunięcie wydarzenia
+
+**Przypadki testowe scenariusza**:
+
+
+| ID i Nazwa Przypadku Testowego           | Warunki wstępne                                                                                 | Dane testowe                                                                                                           | Kroki reprodukcji                                                                                                                                                                                                                                                                                                                               | Oczekiwany rezultat                                                                     |
+|------------------------------------------|-------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| TC2.1 Utworzenie poprawnego wydarzenia   | 1. Istnieje konto organizatora<br>2. Organizator jest zalogowany<br>3. Istnieje dowolna lokacja | 1. NAZWA: "event1"<br>2. OPIS: "description"<br>3. NAZWA BILETU: "standard"<br>4. CENA: 100<br>5. LICZBA BILETÓW: 1000 | 1. Kliknij przycisk dodawania wydarzenia<br>2. Wprowadź nazwę i opis<br>3. Wprowadź dowolną datę początkową i datę końcową o 1 dzień późniejszą<br>4. Wprowadź dowolne godziny początkowe i końcowe <br>5. Wybierz lokację z listy<br>6. Wybierz dowolny obraz<br>7. Zmień nazwę biletu<br>8. Ustaw cenę i ilość<br>9. Kliknij przycisk "Dodaj" | 1. Utworzenie wydarzenia<br>2. Wyświetlenie komunikatu o pomyślnym utworzeniu           |
+| TC2.2 Data końcowa przed początkową      | 1. Istnieje konto organizatora<br>2. Organizator jest zalogowany                                | 1. NAZWA: "event1"<br>2. LICZBA BILETÓW: 10                                                                            | 1. Kliknij przycisk dodawania wydarzenia<br>2. Wprowadź dowolną datę początkową i datę końcową o 1 dzień wcześniejszą<br>3. Ustaw ilość biletów<br>4. Kliknij przycisk "Dodaj"                                                                                                                                                                  | 1. Wyświetlenie komunikatu "Data końcowa nie może być wcześniejsza niż data początkowa" |
+
+<div style="page-break-after: always;"></div>
```

Comment:
to tutaj żeby PDF się generował? czy o co chodzi z tym zakończeniem tabelki tu

Reply:
Dokładnie tak. Plugin do mkdocs kompletnie rozwala format, kiedy tabelki są zbyt długie. Sztuczki z external css-em do mkdocs nie działają:
```
table {
    page-break-inside: avoid;
}

table tr {
    page-break-inside: avoid;
    page-break-after: auto;
}

table td {
    page-break-inside: avoid;
    page-break-after: auto;
}
```